staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -57,13 +67,18 @@ func (a *AdmissionOptions) AddFlags(fs *pflag.FlagSet) {
 		"File with admission control configuration.")
 }
 
-// ApplyTo adds the admission chain to the server configuration
-// the method lazily initializes a generic plugin that is appended to the list of pluginInitializers
+// ApplyTo adds the admission chain to the server configuration.
+// In case admission plugin names were not provided by a custer-admin they will be prepared from the recommended/default values.
+// In addition the method lazily initializes a generic plugin that is appended to the list of pluginInitializers
 // note this method uses:
 //  genericconfig.LoopbackClientConfig
 //  genericconfig.SharedInformerFactory
 //  genericconfig.Authorizer
 func (a *AdmissionOptions) ApplyTo(serverCfg *server.Config, pluginInitializers ...admission.PluginInitializer) error {
+	if len(a.PluginNames) < 0 {
```

Comment:
len < 0?

Reply:
ups, should be len <= 0

Reply:
how about == 0? :)

Reply:
yeah, it would be much easier if only the build in `len` function returned unsigned int. The length of a string cannot be less than 0, it doesn't make sense. You're probably right `<=0` is too defensive.